requirements.in

```diff
@@ -28,3 +28,4 @@ XlsxWriter==3.1.*
 SQLAlchemy>=2.0.9<=2.1.0
 Flask-SQLAlchemy==3.0.*
 psycopg2-binary
+pydot==1.4.2  # ERD diagram
```

Comment:
Maybe we want this only in the dev requirements?

Reply:
Yeah I suppose the CLI command will never be called on a live system.. I'll add a try except block to the command to handle it if somebody tries